ci_config.json

```diff
@@ -65,6 +65,10 @@
     "_comment": ["libmicrohttpd is not available with MSVC CI"],
     "linux_only": true
   },
+  "libobsd": {
+    "_comment": ["Doesn't build with MSVC"],
```

Comment:
e.g. requires POSIX?

Reply:
No, it requires GCC's `#include_next` preprocessor directive. That one is provided by clang-cl but not by MSVC under Visual Studio.

That's why I thought I'd need a new stable, I happened to port the program to Visual Studio after releasing the first version, but it seems ci_config.json doesn't allow specifying Windows-specific command-line options.